stm32wba: Update of `west blobs fetch` command due to field `name` definition in hal_stm32 module.yml

erwango opened the pull request:
stm32wba: Update of west blobs fetch command due to field name definition in hal_stm32 module.yml

erwango committed:
boards: nucleo_wba55cg: Update blob fetch command

Following addition of a `name` to hal_stm32 module, the command to fetch
blobs that are required to build BLE applications on STM32WBA socs is now
changed from `west blob fetch stm32` to `west blob fetch hal_stm32`.
Update documentation and warn users in migration guide.

Signed-off-by: Erwan Gouriou <erwan.gouriou@st.com>

dkalowsk reviewed (dismissed):
Changes look fine, minor question if this needs to also be added to the 4.0 release notes or if the migration guide is enough.

jhedberg commented:
@erwango looks like you need to rebase

erwango committed:
west.yml: hal_stm32: Add name to module.yml

hal_stm32 was missing a `name` in module.yml
Point to the module version where this is fixed.

Signed-off-by: Erwan Gouriou <erwan.gouriou@st.com>

erwango commented:
> Changes look fine, minor question if this needs to also be added to the 4.0 release notes or if the migration guide is enough.

Good question.
This change is quite limited and doesn't bring much value a part compliance. Though, it will impact the existing wba blob users which will get the info in the migration guide. 
In release notes, I'd rather mention new features or changes which bring additional value to users... So not this change.
